src/main/java/org/cbioportal/cdd/service/internal/ClinicalAttributeMetadataPersistentCache.java

```diff
@@ -64,18 +55,19 @@ public CacheManager getCacheManager(String ehcacheXMLFilename) throws Exception
     // retrieve cached TopBraid responses from default EHCache location
     @Cacheable(value = "clinicalAttributeMetadataEHCache", key = "#root.target.CLINICAL_ATTRIBUTES_METADATA_CACHE_KEY", unless = "#result==null")
     public ArrayList<ClinicalAttributeMetadata> getClinicalAttributeMetadataFromPersistentCache() {
-        return clinicalAttributesRepository.getClinicalAttributeMetadata();
+        return clinicalAttributeRepository.getClinicalAttributeMetadata();
     }
 
     @Cacheable(value = "clinicalAttributeMetadataOverridesEHCache", key = "#root.target.OVERRIDES_CACHE_KEY", unless = "#result==null")
     public Map<String, ArrayList<ClinicalAttributeMetadata>> getClinicalAttributeMetadataOverridesFromPersistentCache() {
-        return clinicalAttributesRepository.getClinicalAttributeMetadataOverrides();
+        return clinicalAttributeRepository.getClinicalAttributeMetadataOverrides();
     }
 
     // retrieve cache TopBraid responses from backup EHCache location (and re-populate default EHCache locatin)
     @Cacheable(value = "clinicalAttributeMetadataEHCache", key = "#root.target.CLINICAL_ATTRIBUTES_METADATA_CACHE_KEY", unless = "#result==null")
     public ArrayList<ClinicalAttributeMetadata> getClinicalAttributeMetadataFromPersistentCacheBackup() throws Exception {
         CacheManager backupCacheManager = getCacheManager("ehcache_backup.xml");
+        @SuppressWarnings("unchecked")
```

Comment:
These may not be needed. CacheManagers can be instantiated with known Key/Value types to avoid casting warnings.

Reply:
okay with keeping or removing